js/src/logger.ts

```diff
@@ -4184,17 +4184,25 @@ export class Experiment
         }
       }
 
-      const results = await state.apiConn().get_json(
-        "/experiment-comparison2",
-        {
-          experiment_id: await this.id,
-          base_experiment_id: comparisonExperimentId,
-        },
-        3,
-      );
+      try {
+        const results = await state.apiConn().get_json(
+          "/experiment-comparison2",
+          {
+            experiment_id: await this.id,
+            base_experiment_id: comparisonExperimentId,
+          },
+          3,
+        );
 
-      scores = results["scores"];
-      metrics = results["metrics"];
+        scores = results["scores"];
+        metrics = results["metrics"];
+      } catch {
+        console.warn(
```

Comment:
can you print the error we got from the network, for debugging purposes?

Also, coming soon, we're going to have `x-bt-internal-trace-id` on all requests. I dont want to overload this change, but in the near future, maybe we shoudl refactor all such errors to include that reference.